src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -387,7 +387,7 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "PUT" }
     ),
-    usersLoader: gravityLoader("users"),
+    usersLoader: gravityLoader("users", {}, { headers: true }),
```

Comment:
Annoying but: did you double check in the v1 schema that nothing uses this loader? It won't show up on project-wide find.

Reply:
Yup, I did